bin/setup-local-env.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Exit if any command fails
+set -e
+
+# Change to the expected directory
+cd "$(dirname "$0")/../docker"
+
+# Launch the WordPress docker
+docker-compose up -d
+
+# Wait until the docker containers are setup properely
+echo "Attempting to connect to wordpress"
+until $(curl -L http://localhost:8888 -so - | grep -q "WordPress"); do
+    printf '.'
+    sleep 5
```

Comment:
How long does the setup usually take? Seems like we could check more frequently than once every five seconds, to help reduce time to complete the full test run.

Reply:
It took like 20 seconds for me